### PR TITLE
Updated .md files and exchange.py based on suggestions

### PR DESCRIPTION
**Pull request by K-Mertyris**
Thanks to @tallguyjenks for the suggestions.

### CODE REVIEW

File: Exercise_2_Currency_Ex/exchange.md [hunk 1]
```diff
@@ -6,16 +6,18 @@ Placing the pseudocode in this document will also allow me to keep the codebase
 
 ## Pseudocode
 
-`def estimate_value(budget, exchange_rate):`  
-`def get_change(budget, exchanging_value):`  
-`def get_value(denomination, number_of_bills):`  
-`def get_number_of_bills(budget, denomination):`  
+```Python
+def estimate_value(budget, exchange_rate):
+def get_change(budget, exchanging_value):
```
Comment (tallguyjenks) [inline Exercise_2_Currency_Ex/exchange.md:11]: missed some floating back ticks on the end of this line, either that or its floating spaces, either should be removed

**Review by tallguyjenks — COMMENTED** [review]
just a whitespace fix :)

File: Exercise_2_Currency_Ex/exchange.md (hunk 1 above)
Comment (K-Mertyris) [reply Exercise_2_Currency_Ex/exchange.md:11]: Got it, for some reason the linter didn't catch it. I'll have to turn on the visibility for all characters b/c I can't see stuff like that w/o it. Thanks!

Comment (tallguyjenks) [reply Exercise_2_Currency_Ex/exchange.md:11]: I do the same :) I have all. whitespace visible I believe :) there's also some browser extensions fo github I use to also make that stuff easy to see :D

**Pull request merged**